packages/round-manager/src/features/round/RoundApplicationForm.tsx

```diff
@@ -22,9 +25,17 @@ import ErrorModal from "../common/ErrorModal";
 import { errorModalDelayMs } from "../../constants";
 import { useCreateRound } from "../../context/round/CreateRoundContext";
 import { datadogLogs } from "@datadog/browser-logs";
-import { CheckIcon, PencilIcon } from "@heroicons/react/solid";
+import { CheckIcon, PencilIcon, XIcon, InformationCircleIcon } from "@heroicons/react/solid";
+import { Switch } from "@headlessui/react";
+import ReactTooltip from 'react-tooltip';
 
 export const initialQuestions: QuestionOptions[] = [
+  {
```

Comment:
wondering if we should leave out the Payout Wallet Address question here?

We can create a "hardcoded" component to display on the UI, since it doesn't need to be interactive
Like this: https://github.com/gitcoinco/grants-round/compare/478--toggle-application-question...shavinac/501-toggles 

Also I think we should sync with Grant Hub team about whether or not we should include the Payout Address question in the application schema. Currently they are hard-coding the question in the Grant Hub form -- so if we add the payout question on the round application schema, then there will be duplicate Payout Address questions on Grant Hub